docs/documentation/cli.md

```diff
@@ -1,67 +1,54 @@
 ---
 layout: default
-title: CLI
+title: Soda SQL CLI commands
 parent: Documentation
 nav_order: 2
 ---
 
-# Soda's Command Line Interface (CLI)
+# Soda SQL CLI commands
 
-> See our [Installation guide]({% link getting-started/installation.md %}) on how to install the `soda` command.
+| Command | Description |
+| ------- | ----------- |
+| `soda analyze` | Analyzes the contents of your warehouse and automatically prepares a scan YAML file for each table. Soda SQL puts the YAML files in the `/tables` directory inside the warehouse directory. See [Create a scan YAML file]({% link documentation/scan.md %}#create-a-scan-yaml-file) for details.|
+| `soda create` | Creates a new `warehouse.yml` file and prepares credentials in your `~/.soda/env_vars.yml`. Soda SQL does not overwrite or remove and existing environment variables, it only adds new. See [Create a warehouse YAML file]({% link documentation/warehouse.md %}#create-a-warehouse-yaml-file) for details. |
```

Comment:
Would it make sense to change the order?  users typically do a `soda create` first, then a `soda analyze` and then `soda scan`s